python_modules/libraries/dagster-sigma/dagster_sigma_tests/test_asset_specs.py

```diff
@@ -79,6 +81,27 @@ def test_load_assets_organization_data(sigma_auth_token: str, sigma_sample_data:
         assert len(responses.calls) == calls
 
 
+@responses.activate
+def test_rebuild_workbooks(
```

Comment:
[2] Just flagging that the name of the test and python script would change if we change the factory name.